src/apps/vpn/ui/screens/settings/ViewSubscriptionManagement/SubscriptionManagementItem.qml

```diff
@@ -236,7 +236,7 @@ ColumnLayout {
             objectName: _objectName + "-relayUpsell-upgradeButton"
 
             onClicked: {
-                VPNGleanDeprecated.recordGleanEvent("bundle_upsell_upgrade_clicked");
+                MZGleanDeprecated.recordGleanEvent("bundle_upsell_upgrade_clicked");
```

Comment:
If we are going to rename it to mzglean, maybe we should also rename the rust crate.